packages/babel-parser/test/fixtures/core/uncategorised/361/input.js

```diff
@@ -1 +0,0 @@
-x\
```

Comment:
Deleted this test because it is identical to `esprima/invalid-syntax/migrated_0032`.